### ISSUE TITLE
infrared image implementation

### ISSUE THREAD

**Issue by jasony93** [hysts/pytorch_mpiigaze#60]
First of all, I appreciate for this great work.
I went through the mpiifacegaze / mpiigaze datasets, but I decided that it is hard to add or customize data.
I wanted to train with grayscale images (which will eventually become infrared images).

Is there a simple way to train with grayscale images?

Also, how can I add more data to existing dataset?

**Comment by hysts** [comment]
Hi, @jasony93 
As the images in the MPIIGaze dataset are grayscale, I think you can train a MPIIGaze model with this repo as is if you prepare your dataset in the same format as MPIIGaze dataset.
And I don't think it's that hard to train a grayscale version of the MPIIFaceGaze model. It should work if you change the number of the first channel of the model to 1 and convert input images to grayscale.

**Comment by jasony93** [comment]
Thank you for the quick response. 
I was looking at the code from gaze_estimator.py, and according to following code the image is 3-channels.
And I assumed that the dataset is also 3 channels.

Could you please explain how below code is 3 channels and the dataset is grayscale?

This code is actually from pytorch_mpiigaze_demo.

`@torch.no_grad()
    def _run_mpiifacegaze_model(self, face: Face) -> None:
        image = self._transform(face.normalized_image).unsqueeze(0)

        device = torch.device(self._config.device)
        image = image.to(device)
        prediction = self._gaze_estimation_model(image)
        prediction = prediction.cpu().numpy()`

**Comment by jasony93** [comment]
Thank you I will take a look at it again!
